content/operate/rc/databases/rdi/setup.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: Prepare source database
+alwaysopen: false
+categories:
+- docs
+- operate
+- rc
+description: null
+hideListLinks: true
+weight: 1
+---
+
+## Create new data pipeline
+
+1. In the [Redis Cloud console](https://cloud.redis.io/), go to your target database and select the **Data Pipeline** tab.
+1. Select **Create data pipeline**.
+1. Select your source database type. The following database types are supported:
+    - MySQL
+    - Oracle
+    - SQL Server
+    - PostgreSQL
+1. If you know the size of your source database, enter it into the **Source dataset size** field.
+
+## Prepare source database
+
+Before using the pipeline, you must first prepare your source database to use the Debezium connector for change data capture (CDC).
+
+See [Prepare source databases]({{<relref "/integrate/redis-data-integration/ingest/data-pipelines/prepare-dbs/">}}) to find steps for your database type.
```

Comment:
Just so you are aware, I've changed the structure of the RDI docs a bit recently. Ingest is now known as just RDI, so it has been moved up a level and the ingest folder is no longer part of the pathname.
```suggestion
See [Prepare source databases]({{<relref "/integrate/redis-data-integration/data-pipelines/prepare-dbs/">}}) to find steps for your database type.
```